### PR TITLE
Added Instagram Widget to Social Media Section

### LINKED ISSUE #239: Instagram Feed missing from widgets
**Actual Behaviour**

FB and Twitter

**Expected Behaviour**

Add insta also

**Steps to reproduce it**

^^

**Would you like to work on the issue?**

Open to all

### PR DESCRIPTION
**Pull request by Ryan10145**
- [X] Included a Preview link and screenshot showing after and before the changes.
- [X] Included a description of the change below.
- [X] Squashed the commits.

# Changes done in this Pull Request

- [Preview Link](https://ryan10145.github.io/gci18.fossasia.org/)

- Fixes #239 

## Description / Changes

- Adds in an Instagram widget to the social media section that updates with the latest images from https://www.instagram.com/fossasia/?hl=en

### Before:

image

### After:

image
image
image

- - - - - - - - - - - -

### CODE REVIEW

**Comment by Ryan10145** [comment]
@sarnava1 I have fixed the conflicts.

**Comment by Ryan10145** [comment]
@sarnava1 I've fixed the issues and updated the preview link and the pictures.

**Comment by sarnava1** [comment]
LGTM. Please have a look @abishekvashok @mariobehling @tabesin @meets2tarun

File: index.html [hunk 1]
```diff
@@ -197,6 +197,14 @@ <h1>Connect with Fossasia on Social Media</h1>
                         frameborder="0" allowfullscreen>
                     </iframe>
                 </div>
+                <div class="col-xs-12 col-sm-6 col-md-4 h-100 social-media-widget">
+                    <a href="https://www.instagram.com/fossasia/" class="social-media-title">Fossasia on Instagram</a>
```
[the remark this answers is not in the document]
Comment (simarsingh24) [reply index.html]: doesnt matter in URL

[the remark this answers is not in the document]
Comment (nikit19) [reply index.html]: I am talking about the text

**Comment by abishekvashok** [comment]
Task link: https://codein.withgoogle.com/dashboard/task-instances/6665083598929920/

**Pull request merged**